net/atftp/files/atftpd.init

```diff
@@ -6,6 +6,10 @@ PIDFILE=/tmp/run/atftpd.pid
 
 start() {
```

Comment:
Add a local variable there `local enable`

Reply:
Should I then refactor whole package? Since other variables in this script are not using local variables

Reply:
Done